rate-providers/registry.json

```
@@ -182,6 +182,25 @@
             "warnings": [],
             "factory": "",
             "upgradeableComponents": []
+        },
+        "0xc7177B6E18c1Abd725F5b75792e5F7A3bA5DBC2c": {
+            "name": "TruMaticRateProvider",
+            "summary": "safe",
+            "review": "./TruMaticRateProvider.md",
+            "warnings": [
+                "donation"
+            ],
+            "factory": "",
+            "upgradeableComponents": [
+                {
+                    "entrypoint": "0xA43A7c62D56dF036C187E1966c03E2799d8987ed",
+                    "implementationReviewed": "0x2a9fD373Ed3Ce392bb5ad8Ee146CFAB66c9fAEae"
+                },
+                {
+                    "entrypoint": "0xeA077b10A0eD33e4F68Edb2655C18FDA38F84712",
+                    "implementationReviewed": "0xf98864DA30a5bd657B13e70A57f5718aBf7BAB31"
+                }
```

Also include `StakeManager`.
```suggestion
                },
                {
                    "entrypoint": "0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908",
                    "implementationReviewed": "0xbA9Ac3C9983a3e967f0f387c75cCbD38Ad484963"
                }
```